package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "babel-plugin-import-expander",
-  "version": "0.2.2",
+  "version": "0.2.3",
```

Comment:
don't change version in PR, i'll update it after merging it to master. it's a convention.